src/state.rs

```diff
@@ -23,13 +26,26 @@ impl State {
             window_position: (0, 0).into(),
             selected_command: select_initial_command(&config, &options),
             selection_path: vec![],
+            form_command_task_variables: HashMap::new(),
             config,
             options,
         };
         return state;
     }
 }
 
+pub fn init_variables_form_input(form_command_task_variables: &mut HashMap<String, String>, variables: &Vec<CommandTaskVariable>) {
```

Comment:
statt `&Vec<T>` ist eigentlich immer `&[T]` sinnvoller

Reply:
Well, thats a bit unintuitive O.o

Reply:
why? you can always get a `&[T]` from a vec, and if you just have a shared reference you can't actually do much more with the vec than deref it to a slice